### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by abdellani**

### CODE REVIEW

File: src/components/BookList.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          <th>
+            categorie
+          </th>
+          <th>
+            action
+          </th>
+        </tr>
+        {books.map((book) =>
+          <Book
+            book={book}
```
Comment (SunnyAsar) [inline src/components/BookList.jsx]: Hi, let's give each book a unique key!
- and some semantics here, use tbody

File: src/components/BooksForm.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    super(props);
+    this.state = {
+      title: '',
+      categorie: categories[0]
+    }
+  }
+  handleChange({ title, categorie }) {
+    if (title) {
+      this.setState({ title: title.value })
+    }
```
Comment (SunnyAsar) [inline src/components/BooksForm.jsx]: Hello, this is fine, but let me suggest a simpler implementation and of course this is OPTIONAL.
```
handleChange = (e) => {
    this.setState({
      [e.target.name] : e.target.value
    })
  }
```
and the input can be:

```
<input
          type="text"
          //onChange={(e) => this.handleChange({ title: e.target })}
          onChange={this.handleChange}
          name="title"
          value={this.state.title}
        />
```
this implementation is using the name to track the event and update the state.
- this way no matter the amount of fields you have, the handleChange remains the same, you just have to give the input a name that matches what you have in store for it.

Comment (SunnyAsar) [reply src/components/BooksForm.jsx]: and if you don't like that approach, there is no need to have multiple `setState`, it should look something like this:
```
handleChange(event) {
  const name = event.target.name;

  this.setState({
    title: event.target.value
    [name]: event.target.value
  })
}
```

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @abdellani  and @EmirVatric 👋 
#### Nice one !
- let's get the key for books in place and clean our handleChange

**Request another review when you are ready!** Cheers

**Review by SunnyAsar — APPROVED** [review]
# Hey @abdellani  and @EmirVatric  
- Looking Good now. 
But why are you making all the changes @abdellani 

### You May Pass 👍

**Comment by abdellani** [comment]
Hi @SunnyAsar 
Thank you for the reply.
We did the changes with my account, because they didn't require too much time : )

**Pull request merged**